### PR TITLE
LPS-202505 Use custom drag preview for mouse DnD

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarItem.js [hunk 1]
```diff
@@ -52,22 +42,26 @@ function CriteriaSidebarItem({
 			role="menuitem"
```
Comment (veroglez) [inline modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarItem.js:42]: Could we add the `role="menu"` to the parent `ul`?

Also, this pattern has been recently updated. When we have only one action within a `menuItem`, that action is executed from the `menuItem` with space/enter. There is no need for a `button` inside the `menuItem`.

Comment (sandrodw3) [reply modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarItem.js:42]: Weee applying it 🚀

**Comment by veroglez** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:forward 👏

**Comment by Tim-Cao** [comment]
https://github.com/liferay-echo/liferay-portal/pull/14537

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**